frontend/common/src/components/form/MultiSelect/MultiSelectFieldV2.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import SelectFieldV2, {
+  type SelectFieldV2Props,
+} from "../Select/SelectFieldV2";
+
+export type MultiSelectFieldV2Props = Omit<
+  SelectFieldV2Props,
+  "isMulti" | "forceArrayFormValue"
+>;
+
+const MultiSelectFieldV2 = (props: MultiSelectFieldV2Props) => (
```

Comment:
Is this intended to completely replace the existing multiselect?  If so, could you spin off an issue to move all existing usage to this and remove the old one?  If not, could you give this a more descriptive name so devs know when to prefer this version?

Reply:
Created a new ticket to capture this migration: https://github.com/GCTC-NTGC/gc-digital-talent/issues/3384